src/Components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,184 @@
+import { useProducts } from "../../Context/Product-Context/product-context";
+import "./sidebar.css";
+export const Sidebar = () => {
+  const { state, dispatch } = useProducts();
+  const productCategories = ["Bats", "Balls"]
+  return (
+    <div>
+      <aside className="sidebar">
+        <div className="product-filter">
+          <h4>Filter</h4>
+
+          <h3
+            style={{ cursor: "pointer" }}
+            onClick={() => dispatch({ type: "CLEAR" })}
+          >
+            Clear
+          </h3>
+        </div>
+        <div className="filter-by-price">
+          <div className="slider-wrapper">
+            <input type="range" className="slider" min="0" max="100" />
+            <h4 className="slider-value">50</h4>
+          </div>
+        </div>
+        <div className="product-sort">
+          <h3 className="filter-feature">Sort</h3>
+          <div className="categories">
+            <label className="input-select">
+              <input
+                type="radio"
+                id="low-to-high"
+                name="SORT_BY"
+                value="LOW_TO_HIGH"
+                checked={state.sortBy !== "LOW_TO_HIGH" ? false : true}
```

Comment:
Don't need to return `true` or `false` ,  `!==` operator doing the same

Reply:
Yes , I'll do that 
Thanks for the suggestion